tests/documents/proofOfAddress/proofOfAddressV1.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { promises as fs } from "fs";
+import * as path from "path";
+import { expect } from "chai";
+import { dataPath } from "../../apiPaths";
+import { ProofOfAddressV1 } from "../../../src/documents";
+
+describe("Proof of Address V1 Object initialization", async () => {
+  it.only("should load a complete document prediction", async () => {
```

Comment:
Should also try loading an empty document to make sure we handle null values correctly.
For example: https://github.com/mindee/mindee-api-nodejs/blob/main/tests/documents/invoiceSplitter/invoiceSplitterV1.spec.ts#L8